### PR TITLE
NORALLY: extending the revise operation (v1.2.00)

### PR DESCRIPTION
**Pull request by graju256**
- Extending the **revise** operation
  - to revise the bundle as per the schema changes (entities will be migrated from deprecated entity types to newer types) 
- Modified **renew** operation parameters
  - **_--gateway_** parameter is changed from optional to required. This way, there shouldn't be any confusion about renewing the bundle from. 
  - fixed few issues with it in referencing the metadata
- Modified **import** operation parameters
  - removed **_--options.revise_** option parameter. That means, when required, bundle should be revised separately prior to **import** operation.
- Updated the README
  - updated the gateway profile names in the used examples
  - improved the diff operation section
  - updated the revise operation section
  - updated the deprecated entity types section

### CODE REVIEW

File: modules/graphql-entity-reviser.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
+
+const utils = require("./graphman-utils");
+const revisers = [
+    policiesReviser11100(),
```
Comment (uppoju) [inline modules/graphql-entity-reviser.js:4]: is it OK to keep version in the method names, we might need to same revision in future versions also

Comment (graju256) [reply modules/graphql-entity-reviser.js:4]: I expect, we shouldn't change the existing revisers. Instead, we should always append the new revisers to the list.

Comment (graju256) [reply modules/graphql-entity-reviser.js:4]: This is something like DB upgrades.

Comment (uppoju) [reply modules/graphql-entity-reviser.js:4]: OK

**Review by uppoju — APPROVED** [review]

**Pull request merged**